### ISSUE TITLE
Create a docker image with all tools used by this course

### ISSUE THREAD

**Issue by malachig** [griffithlab/rnabio.org#43]
Another handy way to make it easier for students to do this course on their own would be to create a docker image of all tools needed.  

This could either be a "fat" docker image with all tools specified in a single docker file, or it could be a neatly organized collection of individual docker images.  Many of which may already exist.  The first approach would be convenient and easy to teach, but the second approach would be a more "correct" (or "best practices") demonstration of containerization, but 

This would also ensure that we always have a backup to run a problematic tool when we encounter dependency issues during a course.

**Comment by cinquesoto** [comment]
I totally agree! This would allow everyone to work offline (and save money since there won't be a need for AWS).

**Comment by obigriffith** [comment]
This was created and instructions for its use can be found here:
https://rnabio.org/module-00-setup/0000/07/01/Log_into_Compute/#using-docker

It does need updating though

**Comment by obigriffith** [comment]
This has been further updated and is current (as of 2025). Instructions for using it located as above.
